webapp-django/crashstats/status/tests/test_views.py

```diff
@@ -0,0 +1,36 @@
+from nose.tools import eq_, ok_
+
+from django.core.urlresolvers import reverse
+
+from crashstats.crashstats.tests.test_views import BaseTestViews
+from crashstats.status.models import StatusMessage
+
+
+class TestViews(BaseTestViews):
+
+    def test_status_message(self):
+        # Use any URL really.
+        url = reverse('documentation:supersearch_home')
+
+        response = self.client.get(url)
+        eq_(response.status_code, 200)
+        ok_('#status-message' not in response.content)
+
+        status = StatusMessage.objects.create(
+            message='an incident is ongoing',
+            severity='critical',
+        )
+
+        response = self.client.get(url)
+        eq_(response.status_code, 200)
+        ok_('id="status-message' in response.content)
```

Comment:
Because of my previous point that I think you should support MULTIPLE status messages. Then it no longer makes sense with `id="status-message"`. (It might make sense with `id="status-messages"` as a wrapper).

Also, there's a general best practice that using IDs in DOM is bad. 

I try to ALWAYS use `class="..."` instead of `id="..."` even though there's only 1. Who knows, perhaps some day you want to put one in the footer too. 
The exception is when you really want to point out that there can be only one. Kinda like an `assert` statement in non-test python code. It's useless but indicates to the code-reader something of value.